packages/flame/lib/src/geometry/shape.dart

```diff
@@ -46,6 +46,10 @@ abstract class Shape {
   /// applications of [Shape], for example [HitboxShape]
   double parentAngle;
 
+  /// Whether the context that the shape is in has already prepared (rotated
+  /// etc) the canvas before coming to the shape's render method.
```

Comment:
etc can only be used when there are more than two things right? rotated, translated, etc..